mingw-w64-git/PKGBUILD

```diff
@@ -187,7 +187,6 @@ package_git () {
 
   # Git Bash, Git CMD
   install -d -m755 $pkgdir$SHAREDIR
-  install -m755 ../git-for-windows.ico $pkgdir$SHAREDIR
```

Comment:
I fixed the bug already.

I decided to copy the icon file again. It was available for at least several major Git for Windows versions (IIRC every release since v2.5.0), so it is a major regression to simply take the file away without notice. Basically, by putting the file out there, we told people that it is okay to use that icon in their shortcuts, and in v2.10.0, we broke that promise.